src/evo/mnhftx.cpp

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) 2021-2022 The Dash Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <consensus/validation.h>
+#include <evo/mnhftx.h>
+#include <evo/specialtx.h>
+#include <llmq/commitment.h>
+#include <llmq/signing.h>
+
+#include <chain.h>
+#include <chainparams.h>
+#include <validation.h>
+
+#include <string.h>
```

Comment:
This should not be string.h
```suggestion
#include <string>
```